napari/utils/_proxies.py

```diff
@@ -73,6 +92,25 @@ def __getattr__(self, name: str):
             # )
         return self.create(super().__getattr__(name))
 
+    def __setattr__(self, name: str, value: Any):
+        if self._is_private_attr(name):
+            if self._is_called_from_napari():
+                return super().__setattr__(name, value)
+
+            typ = type(self.__wrapped__).__name__
+            self._private_attr_warning(name, typ)
+
+            # raise AttributeError(
+            #     trans._(
+            #         "Private attribute set ('{typ}.{name}') not allowed in this context.",
+            #         deferred=True,
+            #         name=name,
+            #         typ=typ,
+            #     )
+            # )
```

Comment:
I'm not sure why the commented code is in the original (I'd actually love to delete it), but we definitely shouldn't add a repeat of it without an explanation of why commented code is on the main branch.

Reply:
I'm feeling mildly destructive today, so if you also feel like deleting the original commented code, please go ahead.

Reply:
If I correctly remember this is just preparation on moment where we remove private access. Just remove warning and uncomment exception. 

They have a little different text as first is about getting value and second about setting it. 

But maybe proper solution will be prepare one text and move it to one function?

Personally I do not understand why thi text is translated as it is not connected with user action.